ftw/workspace/upgrades/configure.zcml

```diff
@@ -176,4 +176,13 @@
         directory="profiles/2301"
         />
 
+    <!-- 2301 -> 2302 -->
+    <genericsetup:upgradeStep
+        title="Add the fullname of the workspace creators to the searchable text index."
+        source="2301"
+        destination="2302"
```

Comment:
Since ftw.workspace is already in the 3.x version series, I'd bump this to `3000`.
This should have been done earlier though :wink: